source/AI.cpp

```diff
@@ -166,7 +167,7 @@ namespace {
 	{
 		// Lay out the rules for what constitutes a deployable ship. (Since player ships are not
 		// deleted from memory until the next landing, check both parked and destroyed states.)
-		auto isCandidate = [](const shared_ptr<Ship> &ship) -> bool
+		auto isCandidate = [&](const shared_ptr<Ship> &ship) -> bool
```

Comment:
```suggestion
		auto isCandidate = [](const shared_ptr<Ship> &ship) -> bool
```
you are only using ship, no need for auto capture?

Reply:
It was randomly failing a mac build. It might not happen anymore, idk, I'll test it in a bit.